samples/python/tutorial_code/core/mat_mask_operations/mat_mask_operations.py

```diff
@@ -70,7 +68,7 @@ def main(argv):
     dst0 = sharpen(src)
 
     t = (time.time() - t) / 1000
-    print "Hand written function time passed in seconds: %s" % t
+    print ("Hand written function time passed in seconds: %s" % t)
```

Comment:
Please add this line to the beginning of file (before any imports):
```
from __future__ import print_function
```
This will allow to remove space after `print` (for Python 2.x)

Reply:
does it also work for Python 3.x?

Reply:
Yes